core/logging_config.py

```
@@ -0,0 +1,40 @@
+import logging
```

Ruff (0.12.2)

`logging` imported but unused

Remove unused import: `logging`

(F401)